### PR TITLE
Add Terms to Java Arrays

### PR DESCRIPTION
**Pull request by SSwiniarski**
### Description

From Target Keyword list. Added terms to Java Arrays.

### Type of Change

- Adding a new entry
- Editing an existing entry (fixing a typo, bug, issues, etc)

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/java/concepts/arrays/terms/fill/fill.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
+---
+Title: '.fill()'
+Description: 'Fills an array with a specified value.'
+Subjects:
+  - 'Computer Science'
```
Comment (KTom101) [inline content/java/concepts/arrays/terms/fill/fill.md:5]: I think another subject should be added maybe 'Code Foundations'?
```suggestion
  - 'Code Foundations'
  - 'Computer Science'
```

File: content/java/concepts/arrays/terms/sort/sort.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
+---
+Title: '.sort()'
+Description: 'Sorts an array into ascending order.'
+Subjects:
+  - 'Computer Science'
```
Comment (KTom101) [inline content/java/concepts/arrays/terms/sort/sort.md:5]: Same here, like in "fill.md", perhaps add a second subject:
```suggestion
  - 'Computer Foundations'
  - 'Computer Science'
```

**Review by KTom101 — CHANGES_REQUESTED** [review]
@SSwiniarski Added a few changes to this entry.

**Comment by SSwiniarski** [comment]
@KTom101, I implemented your suggestions.

**Review by KTom101 — APPROVED** [review]
@SSwiniarski  Nice! I'll pass this on for a second review.

**Review by Dusch4593 — APPROVED** [review]
@SSwiniarski LGTM! Nothing to add 👍🏻

**Pull request merged**